getting-started/templates/systemlink-values.yaml

```diff
@@ -355,9 +355,13 @@ dashboardhost:
 
     ## This sets the GF_INSTALL_PLUGINS env variable
     ## https://grafana.com/docs/grafana/latest/installation/docker/#install-plugins-from-other-sources
+    ## Uncomment to configure additional plugins. This will override the plugins included by default
```

Comment:
You should use <ATTENTION> tags when providing instructions that the user must follow.

Reply:
Sorry, but could you explain what you mean by "tags"?

Reply:
Damn markdown comments keep remove brackets. The `<ATTENTION>` tags that are used to call attention to parts of this file that require manual review before deployment.